stable/prometheus/values.yaml

```diff
@@ -179,6 +179,8 @@ kubeStateMetrics:
     #   cpu: 10m
     #   memory: 16Mi
 
+  serviceAccountName: prometheus
```

Comment:
same here, this should most definitely be `serviceAccountName: {{ .Values.kubeStateMetrics.serviceAccountName }}`